gillespy2/solvers/numpy/ssa_solver.py

```diff
@@ -57,21 +67,40 @@ def run(self=None, model=None, t=None, number_of_trajectories=1, increment=None,
         """
         Run the SSA algorithm using a NumPy for storing the data in arrays and generating the timeline.
```

Comment:
I think you meant to say something like 'Run the SSA algorithm. Uses a NumPy array to store results and for generating the timeline.' Just missed a word after NumPy.